website/docs/docs/build/metrics.md

```diff
@@ -493,6 +635,60 @@ Functionality for `develop` is only supported in v1.2 and higher. Please navigat
 
 </VersionBlock>
 
+#### Multiple/Derived Metrics with `metrics.develop`
+If you have a more complicated use case that you are interested in testing, the develop macro also supports this behavior. The only caveat is that you must include the raw tags for any provided metric yml that contains a derived metric. Example below:
+
+```
+{% set my_metric_yml -%}
+{% raw %}
+
+metrics:
+  - name: develop_metric
+    model: ref('fact_orders')
+    label: Total Discount ($)
+    timestamp: order_date
+    time_grains: [day, week, month]
+    calculation_method: average
+    expression: discount_total
+    dimensions:
+      - had_discount
+      - order_country
+
+  - name: derived_metric
+    label: Total Discount ($)
+    timestamp: order_date
+    time_grains: [day, week, month]
+    calculation_method: derived
+    expression: "{{ metric('develop_metric') }} - 1 "
+    dimensions:
+      - had_discount
+      - order_country
+
+  - name: some_other_metric_not_using
+    label: Total Discount ($)
+    timestamp: order_date
+    time_grains: [day, week, month]
+    calculation_method: derived
+    expression: "{{ metric('derived_metric') }} - 1 "
+    dimensions:
+      - had_discount
+      - order_country
+
+{% endraw %}
+{%- endset %}
+
+select * 
+from {{ metrics.develop(
+        develop_yml=my_metric_yml,
+        metric_list=['derived_metric']
+        grain='month'
+        )
+    }}
+```
+
+The above example will return a dataset that contains the metric provided in the metric list (derived_metric) and the parent metric (develop_metric). It will not contain some_other_metric_not_using as it is not designated in the metric list or a parent of the metrics included.
```

Comment:
```suggestion
The above example will return a dataset that contains the metric provided in the metric list (`derived_metric`) and the parent metric (`develop_metric`). It will not contain `some_other_metric_not_using` as it is not designated in the metric list or a parent of the metrics included.
```